docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -55,17 +55,20 @@ The preceding examples use type inference for the type being serialized. An over
 
 ### Serialization example
 
-Here's an example class that contains collections and a nested class:
+Here's an example class that contains collection-type properties and a user-defined type:
 
 [!code-csharp[](snippets/system-text-json-how-to/csharp/WeatherForecast.cs?name=SnippetWFWithPOCOs)]
 
+> [!TIP]
+> "POCO" stands for [plain old CLR object](https://en.wikipedia.org/wiki/Plain_old_CLR_object). A POCO is a .NET type that isn't derived from a special base class and has no attributes describing infrastructure concerns, for example, property visibility in a designer.
```

Comment:
@tdykstra Does this look okay to you?

Reply:
Yes - I think the acronym is common enough that the first sentence with the link would be adequate, but an additional explanatory sentence is fine.  However, "special" base class seems vague. If you keep it, maybe rephrase something like "no base class or attributes describing..."